__tests__/articles.test.js

```diff
@@ -68,3 +69,98 @@ describe("GET articles/:article_id", () => {
+    const examplePatch = { inc_votes: 1 };
+    return request(app).patch("/api/articles/1").send(examplePatch).expect(200);
+  });
+  test("PATCH returns updated article", () => {
+    const examplePatch = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(examplePatch)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 101,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("PATCH returns updated article when inc_votes greater than 1", () => {
+    const examplePatch = { inc_votes: 50 };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(examplePatch)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          article_id: 3,
+          title: "Eight pug gifs that remind me of mitch",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "some gifs",
+          created_at: "2020-11-03T09:12:00.000Z",
+          votes: 50,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("PATCH returns updated article when inc_votes is negative", () => {
+    const examplePatch = { inc_votes: -1000 };
+    return request(app)
+      .patch("/api/articles/5")
+      .send(examplePatch)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          article_id: 5,
+          title: "UNCOVERED: catspiracy to bring down democracy",
+          topic: "cats",
+          author: "rogersop",
+          body: "Bastet walks amongst us, and the cats are taking arms!",
+          created_at: "2020-08-03T13:14:00.000Z",
+          votes: -1000,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("Returns 400 if patch body has incorrect syntax", () => {
+    const examplePatch = { votes: -1000 };
+    return request(app)
+      .patch("/api/articles/5")
+      .send(examplePatch)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid PATCH body");
+      });
+  });
+  test("Returns 400 if patch body missing", () => {
+    return request(app)
+      .patch("/api/articles/5")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid PATCH body");
+      });
+  });
```

Comment:
Please add one more test (400) where the the "inc_votes" property is not a number.